client/packages/host/src/components/Sync/Sync.tsx

```diff
@@ -73,6 +73,11 @@ export const Sync: React.FC = () => {
     isLoading,
     onManualSync,
   } = useHostSync();
+  const { data } = useSync.utils.lastSuccessfulUserSync(
```

Comment:
Now that we make two api calls every second, I'm seeing a ton of requests while I'm sitting on the sync page. (currently 2700 and counting)

The polling isn't really necessary for the user sync, as that will only be initiated by (a) login and (b) the sync button. 

What about adding a success callback to the `updateUser` mutation instead, and fetch the `lastSuccessfulUserSync` then? Or use the `mutateAsync` and `await` it, then call the `lastSuccessfulUserSync` api?